libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drilling.spec.ts

```diff
@@ -142,79 +138,6 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
     });
 });
 
-describe("Interaction", () => {
```

Comment:
test failed because cypress lost the session in the method `postDrillDownStepAttributeDF` -> maybe the cause is that the response time is very slow in the backend. 
In this case: If we split the test as your PR, test tag `checklist_integrated_bear` will pass, but test tag `post-merge_integrated_bear` still failed (we don't have the job/pipeline for this tag so we don't see it fails right now)

By the easy trick, I think we should add the session in the method `postDrillDownStepAttributeDF` like as: establishSession(). It should work correct.